src/utils/clmfates_interfaceMod.F90

```diff
@@ -933,6 +938,24 @@ subroutine dynamics_driv(this, nc, bounds_clump,      &
          this%fates(nc)%bc_in(s)%max_rooting_depth_index_col = &
               min(nlevsoil, active_layer_inst%altmax_lastyear_indx_col(c))
 
+         nlevsoil = this%fates(nc)%bc_in(s)%nlevsoil
+         do j = 1,nlevsoil
+            this%fates(nc)%bc_in(s)%tempk_sl(j) = temperature_inst%t_soisno_col(c,j)
+         end do
+
+         call get_active_suction_layers(this%fates(nc)%nsites, &
+             this%fates(nc)%sites,  &
+             this%fates(nc)%bc_in,  &
+             this%fates(nc)%bc_out)
+
+         do j = 1,nlevsoil
+            if(this%fates(nc)%bc_out(s)%active_suction_sl(j)) then
+               s_node = max(waterstatebulk_inst%h2osoi_vol_col(c,j)/soilstate_inst%eff_porosity_col(c,j) ,0.01_r8)
```

Comment:
The 0.01_r8 magic number should likely be made a parameter. But, we should do that later since testing is already in. I'll look at this more carefully, and if I think it's worth doing turn it into an issue.

Reply:
I made this into an issue in #2049.